client/src/pages/demo/components/index.tsx

```diff
@@ -11,7 +13,12 @@ const DemoComponent: React.FunctionComponent = () => {
         <hr className="w-3/4 mx-auto" />
 
         <div className="p-4 border">
-          <Iframe src="https://www.youtube.com/embed/HGl75kurxok" />
+          <h1>Component: Iframe</h1>
```

Comment:
- make this as anchor tag. Please folow https://github.com/framgia/sph-lms/pull/10#discussion_r1123993692

Reply:
I have inquiry about this sir, https://sun-philippine.slack.com/archives/C04P28YUU92/p1677833332755839?thread_ts=1677821810.550219&cid=C04P28YUU92